Source/DafnyCore/Resolver/NameResolutionAndTypeInference/NameResolutionAndTypeInference.cs

```diff
@@ -5852,6 +5852,8 @@ public record ResolveTypeReturn(Type ReplacementType, ExprDotName LastComponent)
       return rr;
     }
 
+    public record MethodCallInformation(IToken Tok, MemberSelectExpr Callee, List<ActualBinding> ActualParameters);
```

Comment:
Was the `ObjectInvariant` method not being used, or should we add it to this?

Reply:
`[ContractInvariantMethod]` is a Noop, so indeed `ObjectInvariant()` was purely for documentation purposes. I don't think we should keep it though.

Reply:
Yeah, it'd be nice to start taking more advantage of those features in this code base.